client/tools/rhncustominfo/rhn-custom-info.py

```diff
@@ -17,9 +17,13 @@
 
 import sys
 import os
-import string
 import getpass
-import xmlrpclib
+
+try:
+    import xmlrpclib
+except ImportError:
+    import xmlrpc.client as xmlrpclib
+    basestring = str
```

Comment:
I believe we can just use 'str' instead of 'basestring' in all places and remove this line.